src/webgl/p5.RendererGL.js

```diff
@@ -1083,6 +1083,11 @@ p5.RendererGL.prototype._setFillUniforms = function(fillShader) {
   if (this._tex) {
     fillShader.setUniform('uSampler', this._tex);
   }
+  fillShader.setUniform('isTint', !!this._tint);
+  if (this._tint) {
+    fillShader.setUniform('uTint', this._tint);
+  }
+
```

Comment:
See the comment in the shader about how this value should be handled. Also, it should likely be reset to `[1, 1, 1, 1]` within `RendererGL.prototype._update` so that tex1 isn't blue in the example below:
```
image(tex1, 0, 0, 100, 100);
tint(0, 0, 255);
image(tex2, 200, 0, 100, 100);
```